### DISCUSSION TITLE
Timed roles information for players

### DISCUSSION THREAD

**Discussion by newtarkprojects** [gameserverapp/Platform#256] [Suggestions]
Hey Max,

Could it be possible to have an option under triggered tasks to be able to tell players how long is left on their timed custom role? 

For example, all my support tiers use timed custom roles and I would like to offer an option for players to be able to find out how much time they have left on a role when they run a chat command or when they connect to a server.

Thank you,
Gem

**Comment by mvaessen** [comment]
Hi Gem,

That's an interesting suggestion!

Technically this should be possible, assuming we create a new action like `Report Custom role time left` that will allows you to set a custom message (like with Send tokens action) with some extra variables to report the time left.

Lets see if there are more people that like to see this added (upvote Gems suggestion if you do)

**Comment by mvaessen** [comment]
Hi, this is now available on the dashboard!
